src/main.rs

```diff
@@ -1,8 +1,21 @@
 use geo::Point;
-use zonebuilder::Params;
-use zonebuilder::clockboard;
+use zonebuilder::{clockboard, Params};
+use clap::{Arg, App};
 
 fn main() {
+    let clockboard: clap::ArgMatches = App::new("My Test Program")
+    .version("0.1.0")
+    .author("zonebuilders")
+    .about("Builds zones")
+    .arg(Arg::with_name("num_vertices")
+             .short("n")
+             .long("number")
+             .takes_value(true)
+             .help("Number of vertices"))
+    .clockboard();
+
+    let num_vertices = matches.value_of("num_vertices").unwrap_or("input.txt");
```

Comment:
`clap` is kind of intense as a library, honestly. In a few minutes of poking at the documentation, I didn't see an example of how to specify that `num_vertices` is a `usize`, so we can put it in `Params`.

Also, since `zonebuilder` is going to be used both as a library by other code and as a CLI tool, I'm not sure how to nicely deal with the parameters. The current setup we have using `std::default` is nice for libraries. All of the CLI parsing crates seem to also have their own way of specifying optional arguments with default parameters. We don't want to repeat things.

If you want to try `clap`, this PR is fine, and the next step is to probably express that `num_vertices` is `Option<usize>`, and if it is specified, override the default value in `Params`. Another option is to shop around and try some other CLI crates;  https://crates.io/crates/structopt is also popular, and the initial example is a bit more usable-looking, IMO.